### PR TITLE
ALL : Add and Update links

### PR DESCRIPTION
**Pull request by BellezaEmporium**

### CODE REVIEW

File: streams/ru.m3u [hunk 1]
```diff
@@ -370,8 +366,8 @@ https://sc.id-tv.kz/o.m3u8
 http://ott-cdn.ucom.am/s62/index.m3u8
 #EXTINF:-1 tvg-id="OhotnikirybolovInternational.ru" status="blocked",Охотник и рыболов International (1080p) [Geo-blocked]
 https://strm.yandex.ru/kal/ohotnik/ohotnik0.m3u8
-#EXTINF:-1 tvg-id="Olala.ru" status="timeout",O-la-la (576p) [Not 24/7]
-http://194.9.27.164:8103/play/O_la_la/index.m3u8
+#EXTINF:-1 tvg-id="Olala.ru",O-la-la [Not 24/7]
+http://3dff4cc4.ucomist.net/iptv/KY6S3K7AMQ8FQM/9068/index.m3u8
```
Comment (freearhey) [inline streams/ru.m3u:370]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/ru.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/ru.m3u:370]: > image
> 

Going to look for an alternative.

**Review by its0x08 — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
Data linked to Hadi TV was mainly based on website listing. Right now, Channel 8 is streaming in French, while it is indicated that it is on Channel 5.

**Comment by LaneSh4d0w** [comment]
Changed the "O-la-la" link per request.

Proof : https://imgur.com/a/sK9TmsU

**Review by freearhey — APPROVED** [review]

**Review by what-a — APPROVED** [review]

**Pull request merged**